draft-ietf-tls-exported-authenticator.md

```diff
@@ -118,23 +118,32 @@ supporting certificates in the chain. This structure is defined in {{!TLS13}},
 Section 4.4.2.
 
 The certificate message contains an opaque string called
-certificate_request_context which SHOULD be unique for a given connection.  Its format
-is be defined by the application layer protocol and SHOULD be non-zero
-length.  For example, it may be a sequence number used by the higher-level
+certificate_request_context which MAY be set by the application to differentiate
+exported authentictors.  Its format is be defined by the application layer protocol.
+For example, it may be a sequence number used by the higher-level
 protocol during the transport of the authenticator to the other party.  Using
 a unique and unpredictable value ties the authenticator to a given context,
 allowing the application to prevent authenticators from being replayed or precomputed by
 an attacker with temporary access to a private key.
 
 CertificateVerify
-: A signature over the value
-Hash(Handshake Context || Certificate)
-
-This is described in Section 4.2.3 of {{!TLS13}}.  The signature scheme
-MUST be a valid signature scheme for TLS 1.3.  This excludes all RSASSA-PKCS1-v1_5
+: This message is used to provide explicit proof that an endpoint possesses the private key corresponding to its certificate.
+       struct {
```

Comment:
Add a line break here, or the struct will be jammed onto the end of the sentence (which might be the source of your problems).